js/feed.js

```diff
@@ -1,141 +1,213 @@
-(function () {
-  const intervals = [
-    { label: "year", seconds: 31536000 },
-    { label: "month", seconds: 2592000 },
-    { label: "day", seconds: 86400 },
-    { label: "hour", seconds: 3600 },
-    { label: "minute", seconds: 60 },
-    { label: "second", seconds: 0 }
-  ];
+pageToLoad = 1;
 
-  function timeSince(date) {
-    const seconds = Math.floor((Date.now() - date.getTime()) / 1000);
-    const interval = intervals.find(i => i.seconds < seconds);
-    const count = Math.floor(seconds / interval.seconds);
-    return count + " " + interval.label + (count !== 1 ? "s" : "") + " ago";
-  }
+function createActivityFeed(amountOfPosts, loadMoreButton) {
```

Comment:
great, here I would change to the following:

```javascript```
function createActivityFeed(options){
```

`options` would be an object with two properties:

- 'posts`: integer indicating the number of posts
- `loadMore`: boolean indicating whether to show load more button or not